src/filtrering/filtrering-mine-filter/ny_mine-filter-rad.tsx

```diff
@@ -1,34 +1,35 @@
-import {MineFilter} from "../../ducks/mine-filter";
 import {useDispatch, useSelector} from "react-redux";
 import {AppState} from "../../reducer";
 import {finnSideNavn, mapVeilederIdentTilNonsens} from "../../middleware/metrics-middleware";
 import {logEvent} from "../../utils/frontend-logger";
 import {velgLagretFilter} from "../../ducks/filtrering";
-import {apneMineFilterModal} from "../../ducks/mine-filter-ui";
+import {apneMineFilterModal, markerMineFilter} from "../../ducks/lagret-filter-ui-state";
 import {Radio} from "nav-frontend-skjema";
 import RedigerKnapp from "../../components/knapper/rediger-knapp";
 import React from "react";
 import './ny_mine-filter-innhold.less'
 import {ListevisningType} from "../../ducks/ui/listevisning";
+import {LagretFilter} from "../../ducks/lagretFilter";
 
 interface LagretFilterRadProps {
-    filter: MineFilter;
-    filtergruppe: string;
+    lagretFilter: LagretFilter;
+    filtergruppe: ListevisningType;
 }
 
-function NyMineFilterRad({filter, filtergruppe}: LagretFilterRadProps) {
+function NyMineFilterRad({lagretFilter, filtergruppe}: LagretFilterRadProps) {
     const dispatch = useDispatch();
 
-    const valgtMittFilter = useSelector((state: AppState) => filtergruppe === ListevisningType.minOversikt
-        ? state.mineFilterMinOversikt.valgtMineFilter
-        : state.mineFilterEnhetensOversikt.valgtMineFilter);
+    const valgtLagretFilter = useSelector((state: AppState) => filtergruppe === ListevisningType.minOversikt
```

Comment:
valgtMineFilter

Reply:
This should be `valgtMineFilter`, not `valgtLagretfilter` :P 
The last edit was `valgtMittFilter`, which is singular. `valgtMineFilter` is plural

Reply:
It should still be `valgtMineFilter`

Reply:
It is `valgtMineFilter`, just github is showing outdated file.